config/runtime/boot/barebox.jinja2

```diff
@@ -0,0 +1,48 @@
+{%- if boot_commands == 'nfs' and nfsroot is not defined %}
+{%-   set nfsroot = 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/' + debarch %}
```

Comment:
Is there any specific reason to use `20240221.0` images? There were [later updates](https://github.com/kernelci/kernelci-core/blame/main/config/core/rootfs-configs.yaml#L200) to [this rootfs](https://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/)

Reply:
That's just what's currently used in `u-boot.jinja2`. I could add a commit that updates both to e.g. `20250117.0` if that makes sense.

Reply:
@hnez right, that LAVA action template should have been updated, good catch!

Please bump nfsroot defaults in `u-boot.jinja2` as a separate PR and align them in this PR.

Reply:
I've created https://github.com/kernelci/kernelci-core/pull/2787 and updated this PR to use `20250117.0` as well.